### PR TITLE
add mattermost-9

### PR DESCRIPTION
**Pull request by Dentrax**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [X] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [X] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)
- [ ] If non-streamed package names no longer built, open PR to withdraw them (see [WITHDRAWING PACKAGES](https://github.com/wolfi-dev/os/blob/main/WITHDRAWING_PACKAGES.md))

#### For package updates (renames) in the base images

When updating packages part of base images (i.e. cgr.dev/chainguard/wolfi-base or ghcr.io/wolfi-dev/sdk)
- [ ] REQUIRED cgr.dev/chainguard/wolfi-base and ghcr.io/wolfi-dev/sdk images successfully build
- [ ] REQUIRED cgr.dev/chainguard/wolfi-base and ghcr.io/wolfi-dev/sdk contain no obsolete (no longer built) packages
- [ ] Upon launch, does `apk upgrade --latest` successfully upgrades packages or performs no actions

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

File: mattermost-9.yaml [hunk 1]
```diff
@@ -0,0 +1,227 @@
+package:
+  name: mattermost-9
+  version: 9.9.0
+  epoch: 0
+  description: "Mattermost is an open source platform for secure collaboration across the entire software development lifecycle."
+  copyright:
+    - license: MIT
+    - license: Apache-2.0
+    - license: AGPL-3.0-only
```
Comment (imjasonh) [inline mattermost-9.yaml:9]: Are we allowed to put AGPL code in Wolfi?

**Review by imjasonh — CHANGES_REQUESTED** [review]

File: mattermost-9.yaml (hunk 1 above)
Comment (imjasonh) [reply mattermost-9.yaml:9]: Yep!

File: mattermost-9/ENTERPRISE-EDITION-LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+The Mattermost Enterprise Edition (EE) license (the “EE License”)
```
Comment (imjasonh) [inline mattermost-9/ENTERPRISE-EDITION-LICENSE.txt:1]: Why do we have this? It doesn't sound like we're building the enterprise edition.

**Review by jamonation — APPROVED** [review]
Licensing looks good, let's ship it!

**Review by imjasonh — APPROVED** [review]

**Pull request merged**